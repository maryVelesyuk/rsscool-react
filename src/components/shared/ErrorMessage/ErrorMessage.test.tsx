import { describe, it, expect } from "vitest";
import { ErrorMessage } from ".";
import { render } from "@testing-library/react";

describe("testing ErrorMessage", () => {
  it("snapshot", () => {
    const result = render(<ErrorMessage />);
    expect(result).toMatchSnapshot();
  });
});
