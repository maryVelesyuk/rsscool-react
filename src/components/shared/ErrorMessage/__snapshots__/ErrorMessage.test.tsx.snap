// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testing ErrorMessage > snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="error-wrapper"
        data-testid="errorMessage"
      >
        <p>
          Something went wrong!
        </p>
        <img
          alt="Error"
          data-nimg="1"
          decoding="async"
          height="350"
          loading="lazy"
          src="/_next/image?url=%2Ferror.png&w=750&q=75"
          srcset="/_next/image?url=%2Ferror.png&w=384&q=75 1x, /_next/image?url=%2Ferror.png&w=750&q=75 2x"
          style="color: transparent;"
          width="350"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="error-wrapper"
      data-testid="errorMessage"
    >
      <p>
        Something went wrong!
      </p>
      <img
        alt="Error"
        data-nimg="1"
        decoding="async"
        height="350"
        loading="lazy"
        src="/_next/image?url=%2Ferror.png&w=750&q=75"
        srcset="/_next/image?url=%2Ferror.png&w=384&q=75 1x, /_next/image?url=%2Ferror.png&w=750&q=75 2x"
        style="color: transparent;"
        width="350"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
